.github/readme.md

```diff
@@ -12,7 +12,9 @@
 </div>
 
 > **Warning** <br/>
-> It's NOT possible to run any games yet, and there is no ETA for when this will change
+> It's NOT possible to run any games yet, and there is no ETA for when this will change.
+
+> Do not ask for link to games or system files. Piracy is not permitted on the GitHub or in the Discord.
```

Comment:
Shouldn't you use nor here?

Reply:
i will

Reply:
> Shouldn't you use `nor` here?

explain i had a brain fart can you write it out

Reply:
```Do not ask for link to games or system files. Piracy is not permitted on the GitHub nor in the Discord.```

Reply:
all done